tax/tag.go

```diff
@@ -0,0 +1,27 @@
+package tax
+
+import (
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/validation"
+)
+
+// Tag describes a tax tag that can be used to identify additional
+// requirements in an electronic invoice.
+type Tag struct {
```

Comment:
Would it make sense for this to be in `cbc`? It's a pretty generic component and it doesn't include properties that are tax-related. It's primarily used for taxes but I could see this being useful down the line for other types

Reply:
I think it'll make sense as soon as we have a usage example outside of tax. Tags can abused easily, so I've mixed feelings about making them more generic.